web/app.js

```diff
@@ -237,6 +243,10 @@ const PDFViewerApplication = {
       });
     }
 
+    if (AppOptions.get("enableSignatureEditor")) {
+      this.signatureStorage = new SignatureStorage();
+    }
+
```

Comment:
Do we actually need to initialize it this early, or can it be delayed until just before the `SignatureManager`-instance is created below?

*Please note:* Ideally we shouldn't initialize any viewer-components, unless absolutely necessary, before l10n is ready.